docs/Makefile

```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
```

Comment:
I'm not sure a Makefile is really appropriate for this, `sphinx-build` is not a complex command.

Reply:
Yes agreed neither this nor the make.bta above are really needed will remove both.